fe/fe-core/src/main/java/com/starrocks/common/proc/BackendsProcDir.java

```diff
@@ -96,20 +95,7 @@ public ProcResult fetchResult() throws AnalysisException {
     public static List<List<String>> getClusterBackendInfos(String clusterName) {
```

Comment:
Remove the clusterName param?

Reply:
I didn't change the caller logic. I will change it in the following PR.